src/tools/tests/plan-generator-tests.ts

```diff
@@ -196,5 +196,40 @@ describe('recipe2plan', () => {
       assert.lengthOf(keys, 2);
       assert.deepStrictEqual(keys[0], keys[1]);
     });
+    it('creates particles in the same order as the recipe and not the manifest', async () => {
```

Comment:
Why is this important? Order in a recipe should not influence anything

Reply:
There was a minor discussion about it on chat yesterday. You're right, it doesn't influence anything, but we talked about implementing this as a debugging / interpretable generation improvement.